File: api/v2010/feedback_summary/create-default/create-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Converters;
using Twilio.Rest.Api.V2010.Account.Call;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var feedbackSummary = FeedbackSummaryResource.Create(
            startDate: MarshalConverter.DateTimeFromString("2008-01-02"),
            endDate: MarshalConverter.DateTimeFromString("2008-01-02")
        );

        Console.WriteLine(feedbackSummary.Sid);
    }
}

File: api/v2010/feedback_summary/create-default/create-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

feedback_summary = @client.calls
                          .feedback_summaries
                          .create(
                             start_date: Date.new(2008, 1, 2),
                             end_date: Date.new(2008, 1, 2)
                           )

puts feedback_summary.sid
